source/common/network/address_impl.h

```diff
@@ -42,12 +49,15 @@ class InstanceBase : public Instance {
 
   const SocketInterface& socketInterface() const override { return socket_interface_; }
 
+  absl::Status initError() const { return init_error_; }
```

Comment:
per above, just pass `abslStatus&` into the ctor args rather than storing the status in the object.

Reply:
Thanks, will fix.